Civi/Api4/Query/Api4Query.php

```diff
@@ -77,7 +78,12 @@ public function getSql() {
     $this->buildLimit();
     $this->buildGroupBy();
     $this->buildHavingClause();
-    return $this->query->toSQL();
+    $sql = $this->query->toSQL();
```

Comment:
@braders what is this element for?

Reply:
Sorry, this bit was included by accident - I'll remove.